app/uk/gov/hmrc/individualsdetailsapi/connectors/IfConnector.scala

```diff
@@ -108,26 +99,35 @@ class IfConnector @Inject()(servicesConfig: ServicesConfig, http: HttpClient, va
 
   private def recover[A](x: Future[A], emptyResponse: A, apiIfFailedAuditRequest: ApiIfFailureAuditRequest)
                         (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[A] = x.recoverWith {
+      case validationError: JsValidationException => {
+        Logger.error(s"Error parsing IF response: ${validationError.errors}")
```

Comment:
@owen-opencast - I think we need to be careful logging this response to Kibana. When we log schema errors to Kibana the log entry can also contain the data that caused the violation. This can be sensitive data and should not be written to Kibana. Suggest we leave the log entry to indicate something went wrong however remove the message aspect?

Reply:
@owen-opencast  - an alternative would be a to change the logging to debug so we can see in local and development only.